### ISSUE TITLE
Missing files in Hugging Face

### ISSUE THREAD

**Issue by ADHuan** [allenai/satlas#46]
Hi,

I recently downloaded the full Satlas pretrain dataset from [Hugging Face](https://huggingface.co/allenai/satlas-pretrain/tree/main/dataset). However, upon reviewing the file lists, I noticed that several tar files for the NAIP dataset are missing for the years 2013, 2016, 2017, and 2018. The missing files are illustrated in the attached screenshots below.

![2013-1](https://github.com/allenai/satlas/assets/36134153/30bc722f-2575-429a-a08e-8ddbcc977612)
![2013-2](https://github.com/allenai/satlas/assets/36134153/3691b978-e09d-4cb0-b733-f054be37404e)
![2016](https://github.com/allenai/satlas/assets/36134153/e9bf6a60-bdcd-4bad-89a9-7ae2b45799b2)
![2017](https://github.com/allenai/satlas/assets/36134153/8114ee28-bb65-4551-8c1a-d71b50096840)
![2018](https://github.com/allenai/satlas/assets/36134153/0cd8428b-7c36-4f13-83ed-8ef3e24f8d7b)

Additionally, I have roughly calculated the total data size of the available tar files for 2013, 2016, 2017, and 2018.  There seems to be a mismatch between this total and the data size listed in your AWS S3 bucket.

Could you please verify the completeness of the dataset and ensure that all files are available for download on Hugging Face?

**Comment by favyen2** [comment]
Yes it is complete, and I checked the total data size matches for 2018 (1.7 TB). Your screenshots show that files from every year are available. NAIP images are captured once every 2-3 years at a given location.

**Comment by favyen2** [comment]
OK I see now what you mean about missing files. I will check it.

**Comment by favyen2** [comment]
In the meantime please download from S3: https://github.com/allenai/satlas/blob/main/satlaspretrain_urls.txt
